x/crisis/module.go

```diff
@@ -35,17 +38,27 @@ func (AppModuleBasic) RegisterCodec(cdc *codec.Codec) {
 
 // default genesis state
 func (AppModuleBasic) DefaultGenesis() json.RawMessage {
-	return moduleCdc.MustMarshalJSON(DefaultGenesisState())
+	return ModuleCdc.MustMarshalJSON(DefaultGenesisState())
 }
 
 // module validate genesis
 func (AppModuleBasic) ValidateGenesis(bz json.RawMessage) error {
-	var data GenesisState
-	err := moduleCdc.UnmarshalJSON(bz, &data)
-	if err != nil {
-		return err
-	}
-	return ValidateGenesis(data)
+	//TODO
```

Comment:
Not sure if this is already being used but a validation with a TODO seems risky

Reply:
The TODO here is actually just raising awareness of the lack of this `ValidateGenesis` which previously just used a placeholder function (hence functionally the same as returning `nil` you just may be tricked into thinking some validation is actually going on if you just look at `module.go`)

Reply:
Actually completing a validate genesis on the crisis keeper should occur as a separate PR

Reply:
https://github.com/cosmos/cosmos-sdk/issues/4472